scripts/py_matter_yamltests/matter_yamltests/definitions.py

```diff
@@ -257,9 +261,26 @@ def __get_targets_by_cluster_name(self, cluster_name: str, target_type: _ItemTyp
             _ItemType.Struct: self.__structs_by_name,
         }
 
+        global_target_mapping = {
+            _ItemType.Request: None,
+            _ItemType.Response: None,
+            _ItemType.Attribute: None,
+            _ItemType.Event: None,
+            _ItemType.Bitmap: self.__global_bitmaps,
+            _ItemType.Enum: self.__global_enums,
+            _ItemType.Struct: self.__global_structs,
+        }
+
         # The idl parser remove spaces
         cluster_name = cluster_name.replace(' ', '')
-        return target_mapping[target_type].get(cluster_name)
+        global_target = global_target_mapping[target_type]
+        target = target_mapping[target_type].get(cluster_name)
+
+        if target is None:
+            return global_target
+        if global_target is None:
+            return target
+        return target | global_target
```

Comment:
when would this happen?

Reply:
Which "this"?  That a cluster has both global and non-global elements of some type?  I'd say "usually".

Reply:
OH, I was thinking this was per-element, ie an element that was both global and cluster, but yeah, this makes much more sense.